aws/resource_aws_batch_job_queue_test.go

```diff
@@ -68,10 +69,15 @@ func TestAccAWSBatchJobQueue_basic(t *testing.T) {
 			{
 				Config: config,
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckBatchJobQueueExists("aws_batch_job_queue.test_queue", &jq),
+					testAccCheckBatchJobQueueExists(resourceName, &jq),
 					testAccCheckBatchJobQueueAttributes(&jq),
 				),
 			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
```

Comment:
This new acceptance testing is currently failing for this test and the others:

```
--- FAIL: TestAccAWSBatchJobQueue_basic (71.48s)
    testing.go:640: Step 1 error: : Provided name  is not supported pattern[a-zA-Z_0-9-]{1,128}
        	status code: 400, request id: fb70a8c2-e8e2-4417-a94e-7f357fda0ebe
```

This is happening because the resource `Read` function is dependent on the `name` attribute being set:

https://github.com/terraform-providers/terraform-provider-aws/blob/b4a971dfc9db81c20df443b0274f2893a976a30c/aws/resource_aws_batch_job_queue.go#L89

The easiest fix here should be to use the resource ID instead of trying to read the `name` attribute, since the [Batch DescribeJobQueues API](https://docs.aws.amazon.com/batch/latest/APIReference/API_DescribeJobQueues.html#Batch-DescribeJobQueues-request-jobQueues) notes support for full ARN in addition to just the name, e.g.

```go
jq, err := getJobQueue(conn, d.Id())
```